src/panels/lovelace/editor/config-elements/hui-light-card-editor.ts

```diff
@@ -0,0 +1,111 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import { struct } from "../../common/structs/struct";
+import "@polymer/paper-input/paper-input";
+
+import { EntitiesEditorEvent, EditorTarget } from "../types";
+import { hassLocalizeLitMixin } from "../../../../mixins/lit-localize-mixin";
+import { HomeAssistant } from "../../../../types";
+import { LovelaceCardEditor } from "../../types";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { Config } from "../../cards/hui-light-card";
+import { configElementStyle } from "./config-elements-style";
+
+import "../../components/hui-theme-select-editor";
+import "../../components/hui-entity-editor";
+
+const cardConfigStruct = struct({
+  type: "string",
+  id: "string|number",
```

Comment:
I believe this can be taken out now. @bramkragten

Reply:
It should yeah

Reply:
I'll rebase all my editor PRs and verify